src/database/queries/ScheduledItem.ts

```diff
@@ -28,7 +28,9 @@ export async function getScheduledItems(
 
   // Get a flat array of scheduled items from Prisma
   const items = await db.scheduledItem.findMany({
-    orderBy: { scheduledDate: 'asc' },
+    // we need to order by scheduleDate first, as we perform a programmatic
+    // groupBy below on that field
+    orderBy: [{ scheduledDate: 'asc' }, { createdAt: 'asc' }],
```

Comment:
I was wondering if we could use `updatedAt` instead to circumvent the need to add a `manualSort` field.